ibis/expr/operations/vectorized.py

```diff
@@ -8,8 +8,11 @@
 from ibis.expr.operations.reductions import Reduction
 
 
+@public
 class VectorizedUDF(Value):
     func = rlz.instance_of((FunctionType, LambdaType))
+    func_summary = rlz.optional(rlz.string)
+    func_desc = rlz.optional(rlz.string)
```

Comment:
It can help ease the pain of updating existing users if you put these arguments at the end of the declaration.